src/Entity/Event.php

```diff
@@ -0,0 +1,348 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\EventRepository")
+ */
+class Event
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $name;
+
+    /**
+     * @ORM\Column(type="string", length=1000)
+     */
+    private $shortDescription;
+
+    /**
+     * @ORM\Column(type="string",nullable=true)
```

Comment:
@daria-sieroshtan 
I think that short description is optional and long one is mandatory
In case you need hort description but only have long - show wrapped version of long (modify method `getShortDescription()`

Reply:
Thanks for the idea. Currently this way doesn't match to expected info provided from users, but I'll think it over once again later.